ArcGISRuntimeSDKQt_CppSamples/CloudAndPortal/ShowOrgBasemaps/ShowOrgBasemaps.qml

```diff
@@ -175,7 +175,6 @@ ShowOrgBasemapsSample {
             margins: 16 * scaleFactor
         }
         visible: mapView.visible
-        iconSource: "qrc:/Samples/CloudAndPortal/ShowOrgBasemaps/ic_menu_back_dark.png"
```

Comment:
@ldanzinger iconSource was removed as it is no longer available in QtQuick Controls 2 buttons in Qt 5.9, but an icon property was later re-introduced in Qt 5.10. If we are upgrading then iconSource can be added without issue. Ohterwise we'll need to add images within buttons which I wanted to avoid.